app/controllers/trips_controller.rb

```diff
@@ -0,0 +1,51 @@
+class TripsController < ApplicationController
+  def index
+  end
+
+  def new
+
+  end
+
+  def create
+    @trip = Trip.new
+    @trip.passenger_id = params[:passenger_id]
+    @trip.driver_id = Driver.all.sample.id
```

Comment:
Are you assuming all drivers are always available?  

I also notice all new trips are free (i.e. the cost is $0.0).  How generous of you both!